IFComp/root/src/_current_entry_row.tt

```diff
@@ -45,6 +45,8 @@
             Web-based
         [% ELSIF entry.platform == 'tads' %]
             TADS [% INCLUDE guide_link %]
+        [% ELSIF entry.platform == 'tads-web-ui' %]
```

Comment:
Could you please change this from a separate ELSIF into another possible condition that causes 'Web-based' to show up on the ballot? (Coz that's all that the IFComp *player* will care about, at least this year.)

Adding `|| entry.platform == 'tads-web-ui'` after `entry.platform == 'website'` should work.